libs/ui/src/lib/footer/footer.component.scss

```diff
@@ -4,6 +4,7 @@ footer {
   padding: 16px;
   justify-content: space-between;
   align-content: center;
+  border: 1px solid red;
```

Comment:
No need to add border; it looks pretty ugly on the marketplace page because the iframe also has a footer.

Reply:
will remove.